File: python/7 kyu/352.py
# https://www.codewars.com/kata/5751aa92f2dac7695d000fb0/train/python

# Passed

def beasts(heads, tails):    
    hydra = (heads - 2 * tails) / 3
    orthus = tails - hydra
    
    result = [int(orthus), int(hydra)] if hydra >= 0 and orthus >= 0 and\
                                int(hydra) == hydra and int(orthus) == orthus else 'No solutions'
    
    return result

output = beasts(123, 39)
print(output)

File: nodejs/6 kyu/352.js
// https://www.codewars.com/kata/541c8630095125aba6000c00/train/javascript

// Passed

function digitalRoot(n) {
  let digits = String(n)
    .split("")
    .map((strDigit) => Number(strDigit));
  if (digits.length > 1) {
    let newNumber = digits.reduce((prev, curr) => prev + curr, 0);
    return digitalRoot(newNumber);
  }

  let result = digits[0];
  return result;
}

const output = digitalRoot(456);
console.log(output);